sites/friday/src/manage-resources/adjust-resources.md

```diff
@@ -0,0 +1,239 @@
+
+{{< /note >}}
+
+After you've set resources, your environment is redeployed,
+which causes a short downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
+   ![Apps and services tree](/images/flexible-resources/apps-services-tree.png "0.2")
+   A window pops up.
+3. Click **Configure**.
+4. For each app and service, select a CPU & RAM combination, and enter the amount of disk/storage you want to allocate.
+   ![Configure your resources on the current environment window](/images/flexible-resources/configure-flexible-resources.png)
+   Note that the values available in the **CPU & RAM** menus depend on the [container profile](#advanced-container-profiles) of each instance.
+5. Click **Save**.</br>
+   You environment is redeployed, which causes a short downtime.
+
+{{< /codetabs >}}
+
+## Horizontal scaling
+
+For apps and workers, you can also define how many instances you want to deploy.
+To do so, follow these steps:
+
+{{< note >}}
+
+When you have several instances of an app, the {{% vendor/name %}} router randomly distributes requests to available instances.
+
+{{< /note >}}
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+To define how many instances of an app or worker you want to deploy,
+you can use the {{% vendor/name %}} CLI's interactive prompts,
+or run commands manually.
+
+- **Interactive prompts:**
+
+  Run the `{{% vendor/cli %}} resources:set` command, and follow the prompts to set resources for each of your apps and services.
+
+  For further guidance on how to set resources using the CLI, run the `{{% vendor/cli %}} resources:set --help` command.
+
+- **Manual commands:**
+
+  To scale an app or worker, run the following command:
+
+  ```bash
+  {{% vendor/cli %}} resources:set --count {{< variable "APP_NAME" >}}:{{< variable "NUMBER_OF_INSTANCES" >}}
+  ```
+  
+  For example, to scale your `myapp` app to 3 instances, run the following command:
+
+  ```bash
+  {{% vendor/cli %}} resources:set --count myapp:3
+  ```
+
+  You can also set the same instance count for all your apps using a wildcard.
+  To do so, run the following command:
+  
+  ```bash
+  {{% vendor/cli %}} resources:set --count '*:{{< variable "NUMBER_OF_INSTANCES" >}}'
+  ```
+
+  For example, to scale all your apps to 3 instances, run the following command:
+
+  ```bash
+  {{% vendor/cli %}} resources:set --count '*:3'
+  ```
+
+  For further guidance on how to set resources using the CLI, run the `{{% vendor/cli %}} resources:set --help` command.
+
+  After you've set the number of instances for your apps and workers, your environment is redeployed.
+  If you've made no other changes, this redeployment causes no downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
```

Comment:
Same comment as the previous one about the new buttons to open the `Configure resources` page.